src/templates/project/project.tsx

```diff
@@ -21,6 +25,16 @@ const ProjectPage: React.FC<ProjectPageProps> = ({ data }) => {
           </Heading1>
         </SectionContent>
       </Section>
+      <Section backgroundColor={theme.colors.pebble}>
+        <SectionContent>
+          <S.AboutSectionWrapper>
+            <S.DescriptionWrapper>
+              <AboutProject />
```

Comment:
Mohu poprosit o zadefinování mock dat v rámci této komponenty pro celou stránku a odstranění použití `defaultProps`? Ať už jako konstantu nebo rovnou jako props.